package-lock.json

```diff
@@ -31058,33 +31058,6 @@
         }
       }
     },
-    "packages/semantic-conventions": {
-      "name": "@opentelemetry/semantic-conventions",
-      "version": "1.25.1",
-      "extraneous": true,
```

Comment:
@pichlermarc I've seen that in the past too. Note the `"extraneous": true` that (some version of) npm added. I'm not aware of npm or package-lock design docs that clarify these fields. A discourteous guess is that adding "extraneous" was a band-aid added in npm's package-lock handling at some point.

Reply:
https://github.com/npm/cli/blob/latest/workspaces/arborist/README.md is perhaps the best such doc. I haven't spent the time to learn it.